inst/include/child_model_simulation.hpp

```diff
@@ -1413,29 +1549,33 @@ void hc_art_initiation_by_age(int time_step,
     } else {
       intermediate.children.hc_adj(0) =  state_next.children.hc_art_init(0) / intermediate.children.hc_initByAge(0);
     }
-    //divergence is coming from this hc_art_init
 
     for (int s = 0; s <ss.NS; ++s) {
       for (int cat = 0; cat < hc_ss.hcTT; ++cat) {
         for (int a = 0; a < pars.base.options.p_idx_fertility_first; ++a) {
           for (int hd = 0; hd < hc_ss.hc1DS; ++hd) {
+
+
             if ((intermediate.children.hc_adj(0) * cpars.hc_art_init_dist(a, time_step)) > 1.0) {
               intermediate.children.hc_art_scalar(0) = 1.0;
               //issue is that in 2030 this is being activated when it shouldn't be for age one
               //TLDR is that too many people are initiating who shouldn't bev
             } else {
               intermediate.children.hc_art_scalar(0) = intermediate.children.hc_adj(0) * cpars.hc_art_init_dist(a, time_step);
             }
-            if (cpars.hc_art_val(0, time_step) > 0.0) {
+
+            if ((cpars.hc_art_val(0, time_step) + cpars.hc_art_val(0, time_step - 1)) > 0.0) {
```

Comment:
Is this ever called when `time_step` is 0?

Reply:
nope!